cmd/posix_test.go

```diff
@@ -35,7 +36,13 @@ func newPosixTestSetup() (StorageAPI, string, error) {
 	if err != nil {
 		return nil, "", err
 	}
-	posixStorage, err := newPosix(diskPath)
+	var endpoints []*url.URL
+	// Initialize a new posix layer.
+	endpoints, err = parseStorageEndpoints([]string{diskPath})
+	if err != nil {
+		return nil, "", err
+	}
+	posixStorage, err := newPosix(endpoints[0])
```

Comment:
the reason to add that is to bring in support for connecting to a remote RPC disk as well. I haven't added that yet.